Storage/ThirdParty/SwiftData.swift

```diff
@@ -267,6 +267,8 @@ private class SQLiteDBStatement {
             } else if obj is Date {
                 let timestamp = (obj as! Date).timeIntervalSince1970
                 status = sqlite3_bind_double(pointer, Int32(index+1), timestamp)
+            } else if obj is UInt64 {
```

Comment:
Note: Date.now() has always produced a `Timestamp` which is a UInt64 which makes me curious about if this change is needed.